invenio_app_ils/ui/src/invenio_app_ils/common/config/invenioConfig.js

```diff
@@ -1,7 +1,7 @@
 export const invenioConfig = {
   circulation: {
     deliveryMethods: {
-      DELIVERY: 'Delivery',
+      DELIVERY: 'Have it delivered to my office',
```

Comment:
`Deliver it, to my office` instead of the passive voice.

Reply:
Now this triggered in my mind another issue. This should happen only in CDS Books, not in ILS. Delivery should be empty, or just PICKUP.
Same as the discussion of today's stand-up. Maybe @equadon has in mind something to override this config in cds-books.

Reply:
I have a label for this field in the form 'I would like to...' and then the delivery method completes the sentence. `I would like to deliver it, to my office` sounds like you will deliver it yourself, with passive voice it implies that someone else will take care of it. But I'm not an english expert, any suggestions guys @zzacharo @ntarocco @equadon ?

Reply:
I think it's good as it is now. `I would like to have it delivered to my office` sounds neutral.

@ntarocco Regarding the config, how flexible and elegant do we need it to be? We could have the base config as it is now and add a `config` prop to the `App` component. We would then merge the `config` prop with the base config.

Reply:
Joining the party, I think we need probably to decide if we want to pass the config in the react level (as @equadon suggests having a config prop) or in the flask level. In the latter case during build time we could export a `.env` file and build react based on that. Ofc we will need in any case to merge the internal `config.js` file with either option we choose.

Reply:
Let's discuss live :) thanks for thinking about this! :)